src/design-guidelines/color-system/examples/contrast-color-palette.scss

```diff
@@ -1,12 +1,6 @@
 @use '../../../style/functions';
```

Comment:
I've got to send some love for this new look on limel-switch too! 😍 👏

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212078429-ffc5ad6b-8879-4ea1-8669-50de8456835f.png"><img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212078562-a7fee85c-bd3e-4d84-b38b-2064c9676587.png">

(New on the left, if anyone happens to read this who doesn't already know 😊)